expo/gbe/report_views.py

```diff
@@ -173,28 +175,36 @@ def personal_schedule(request, profile_id='All'):
                   {'people': people})
 
 
-def review_act_techinfo(request, show_id=1):
+def review_act_techinfo(request, show_id=None):
     '''
     Show the list of act tech info for all acts in a given show
     '''
-    reviewer = validate_perms(request, ('Tech Crew',))
-
+    validate_perms(request, ('Tech Crew',))
+    conf_slug = request.GET.get('conf_slug', None)
+    conference = get_conference_by_slug(conf_slug)
     # using try not get_or_404 to cover the case where the show is there
     # but does not have any scheduled events.
     # I can still show a list of shows this way.
-    try:
-        show = conf.Show.objects.get(eventitem_id=show_id)
-        acts = show.scheduler_events.first().get_acts(status=3)
-        acts = sorted(acts, key=lambda act: act.order)
-    except:
-        show = None
-        acts = []
+    show = None
+    acts = []
+
+    if show_id:
+        try:
+            show = conf.Show.objects.get(eventitem_id=show_id)
+            acts = show.scheduler_events.first().get_acts(status=3)
+            acts = sorted(acts, key=lambda act: act.order)
+        except:
+            logger.error("review_act_techinfo: Invalid show id")
+            pass
 
     return render(request,
                   'gbe/report/act_tech_review.tmpl',
                   {'this_show': show,
                    'acts': acts,
-                   'all_shows': conf.Show.objects.all()})
+                   'all_shows': conf.Show.objects.filter(
+                       conference=conference),
+                   'conference_slugs': conference_slugs()},
```

Comment:
When the user does the following:
1 - go to http://localhost:8282/reports
2 - select "Act Tech Info"
3 - pick an old conference

Then the conference picker defaults back to the current confernce.  If you send in the conference item, the picker will default show the currently selected conference.